service/src/ocall_bridge/bridge_api.rs

```diff
@@ -178,6 +191,12 @@ pub trait WorkerOnChainBridge {
 	fn send_to_parentchain(&self, extrinsics_encoded: Vec<u8>) -> OCallBridgeResult<()>;
 }
 
+/// Trait for updating metrics from inside the enclave.
+#[cfg_attr(test, automock)]
+pub trait WorkerMetricsBridge {
```

Comment:
The `Worker` prefix follows the naming for all the other o-call bridges we have. We can discuss dropping that prefix, but I'd prefer to have a consistent naming convention (which might not be ideal), instead of having no convention.

Yes, plural and singular is on purpose. `update_metric` is the interface to update a single metric. When it's more general, like the bridge trait, I use the plural.

Reply:
Ok, I see, thanks for the explanation.